sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -1330,18 +1332,15 @@ object QueryCompilationErrors {
       s"Expected: ${dataType.typeName}; Found: ${expression.dataType.typeName}")
   }
 
-  def groupAggPandasUDFUnsupportedByStreamingAggError(): Throwable = {
```

Comment:
We will have to manually throw an exception of `AnalysisException` where `groupAggPandasUDFUnsupportedByStreamingAggError` was referred.

Reply:
Yeah, let me address